great-idea/css/index.css

```diff
@@ -63,4 +65,269 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* Main Container */
+
+.container {
+	width: 1000px;
+	margin: auto;
+	border: 1px solid gray;
+}
+
+/* Header Section Selectors */
+
+header {
+	background-color: lightblue;
+    height: 100px;
+    display: flex;
+    align-items: center;
+	justify-content: space-around;
+}
+
+header nav {
+	font-size: 24px;
+}
+
+header nav a {
+	margin: 12px;
+	text-decoration-line: none;
+    color: black;
+}
+
+/* Services Section Selectors (services html) */
+
+.services-info-img {
+	width: 600px;
+}
+
+section.services {
+	margin-top: 25px;
+}
+
+section.services h1 {
+	font-size: 28px;
+}
+
+/* Call To Action Selectors (index.html) */
+
+section.cta {
+	background-image: url("../img/header-img.png");
+    background-repeat: no-repeat;
+    background-position: 90%;
+    background-size: contain;
+    margin-top: 38px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.cta-text button {
+	font-size: 21px;
+    padding: 9px;
+    width: 206px;
+	background-color: white;
+    color: black;
+	border: 1px solid black;
+	margin-left: 53px;
+    margin-top: 19px;
+}
+	
+.cta-text {
+	margin-left: 84px;
+}
+
+section.cta div h1 {
+	font-size: 80px;
+    text-align: center;
+}
+
+/* Main Content Selectors (index.html) */
+
+section.main-content {
+	background-image: url("../img/mid-page-accent.jpg");
+    background-repeat: no-repeat;
+    background-position: 42% 42%;
+}
+
+
+/* Top/Bottom Content Selectors */
+
+.top-content {
+	display: flex;
+}
+
+.bottom-content {
+	display: flex;
+}
+
+.bottom-content .text-content {
+	margin-right: 35px;
+}
+
+div.bottom-content {
+	margin-top: 235px;
```

Comment:
this margin-top would also work under LINE 161 without needing this extra selector

Reply:
Man - looking back at that I have no idea why I did that - smh!